pipeline-automation-scripts/pipeline_automatation_uc1/readme.md

```diff
@@ -0,0 +1,21 @@
+# CDF Pipeline automation python script
+
+This python script creates, deploy and run CDF data pipeline from exported json.
+
+Python script creates 3 different compute profiles named 12wn_profile,25wn_profile and 50wn_profile.Compute profiles are created from 
+compute_profile_1.json,compute_profile_2.json,compute_profile_3.json these json files.
+
+Download Mssql-server jdbc driver for database plugin from [here](https://docs.microsoft.com/en-us/sql/connect/jdbc/download-microsoft-jdbc-driver-for-sql-server?view=sql-server-ver15)
+
+**To run Python script use following command :** 
```

Comment:
Are there any prerequisites before running this script?

Reply:
Added prerequisites in readme.md file.